src/source/meta.scala

```diff
@@ -57,7 +57,9 @@ object MExtern {
     boxed: String, // Java typename used if boxing is required, must be an object.
     reference: Boolean, // True if the unboxed type is an object reference and qualifies for any kind of "nonnull" annotation in Java. Only used for "record" types.
     generic: Boolean, // Set to false to exclude type arguments from the Java class. This is should be true by default. Useful if template arguments are only used in C++.
-    hash: String // A well-formed expression to get the hash value. Must be a format string with a single "%s" placeholder. Only used for "record" types types with "eq" deriving when needed.
+    hash: String, // A well-formed expression to get the hash value. Must be a format string with a single "%s" placeholder. Only used for "record" types types with "eq" deriving when needed.
+    writeToParcel: String, // A well-formed expression to write value into android.os.Parcel. Must be a format string with a single "%s" placeholder. Only used for "record" types types
```

Comment:
I'm not familiar enough with the external type / YAML support.  Are these new fields considered optional? What will happen if they're missing?  My main concern is that this new feature shouldn't break any existing users with yaml files who aren't depending on this new feature.  It's fine for users to have to add these new fields as soon as they want to derive parcelable.